R/filterFeatures.R

```diff
@@ -150,9 +150,7 @@ filterFeatures = function(task, method = "randomForestSRC_importance", fval = NU
     # Set the the filter values of the mandatory features to infinity to always select them
     fval[fval$name %in% mandatory.feat, "value"] = Inf
   }
-  if (select == "threshold") {
-    nselect = sum(fval[["value"]] >= threshold, na.rm = TRUE)
-  }
```

Comment:
This is a separate issue, isn't it?

Reply:
It could be considered a separate issue, but it is part of the problem with ensemble filters not working correctly, which is why I included it. Happy to do a separate PR if you like.

Reply:
Please see my comment in Issue #2685 regarding this.